app/models/email.rb

```diff
@@ -2,6 +2,7 @@ class Email < ApplicationRecord
   # Note that most, but not all, emails that we send are in regards to an
   # application form. For sign-ups and sign-ins application_form will be nil.
   belongs_to :application_form, optional: true
+  has_many :email_clicks
```

Comment:
Just double checking ... are the email records never destroyed? We don't need `dependent: destroy`, do we?